src/modules/storage.js

```diff
@@ -23,17 +23,17 @@ class Storage {
 
   listRemove(taskIndex) {
     this.taskList = this.taskList.filter((task) => task.index !== taskIndex);
-
     // eslint-disable-next-line no-return-assign
     this.taskList.forEach((task, index) => task.index = index);
-
     localStorage.setItem('tasks', JSON.stringify(this.taskList));
   }
 
-  // listRemoveCompleted() {
-  //   this.taskList = this.taskList.filter((task) => !task.completed);
-  //   localStorage.setItem('tasks', JSON.stringify(this.taskList));
-  // }
+  listRemoveCompleted() {
+    this.taskList = this.taskList.filter((task) => !task.completed);
+    // eslint-disable-next-line no-return-assign
+    this.taskList.forEach((task, index) => task.index = index);
+    localStorage.setItem('tasks', JSON.stringify(this.taskList));
+  }
```

Comment:
- [OPTIONAL] I think it's not a good practice to disable `Eslint Linter` using inline comments 🤷‍♂️
> Hint 😉: to fix this eslint error `Arrow function should not return assignment` just add a curly parentheses `{ }` 
> For Example 

```
    this.taskList.forEach((task, index) => {
      task.index = index;
    });
```

Reply:
Thank you for the suggestion. it seems that I did the implement that soon after in this commit. https://github.com/mohamedSabry0/todo-list-2/commit/f0da280efdd5abea4d0c73d903990b1a40eac82c

I realize that I should have just pushed it to this branch instead. I'm not sure what happened back then but I'll make sure to avoid working on the development directly even for small edits.